perftestnotebook.py

```diff
@@ -51,10 +54,14 @@ def __init__(self, file_groups, config, custom_transform=None):
                 obj = getattr(module, name)
                 if isinstance(obj, type) and issubclass(obj, tfmr.Transformer):
                     self.transformer = obj(files=[])
-                    logger.info("Found %s transformer" % self.transformer.__class__.__name__)
+                    logger.info(
```

Comment:
It seems like you might be using the wrong black formatting - can you make sure you run black but with a max line length of 100?